<ui version="4.0" >
 <class>MediaListForm</class>
 <widget class="QWidget" name="MediaListForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Media Tree</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QTreeWidget" name="mp_treeWidget" />
   </item>
  </layout>
  <action name="actionRefreshMediaList" >
   <property name="icon" >
    <iconset resource="../main.qrc" >:/images/view-refresh.png</iconset>
   </property>
   <property name="text" >
    <string>Refresh Media List</string>
   </property>
   <property name="statusTip" >
    <string>Requery the director for the list of media.</string>
   </property>
  </action>
  <action name="actionEditVolume" >
   <property name="icon" >
    <iconset resource="../main.qrc" >:/images/cartridge-edit.png</iconset>
   </property>
   <property name="text" >
    <string>Edit Volume</string>
   </property>
  </action>
  <action name="actionListJobsOnVolume" >
   <property name="icon" >
    <iconset resource="../main.qrc" >:/images/emblem-system.png</iconset>
   </property>
   <property name="text" >
    <string>List Jobs On Volume</string>
   </property>
  </action>
  <action name="actionDeleteVolume" >
   <property name="icon" >
    <iconset resource="../main.qrc" >:/images/edit-delete.png</iconset>
   </property>
   <property name="text" >
    <string>Delete Volume</string>
   </property>
  </action>
  <action name="actionPruneVolume" >
   <property name="icon" >
    <iconset resource="../main.qrc" >:/images/edit-cut.png</iconset>
   </property>
   <property name="text" >
    <string>Prune Volume</string>
   </property>
  </action>
  <action name="actionPurgeVolume" >
   <property name="icon" >
    <iconset resource="../main.qrc" >:/images/weather-severe-alert.png</iconset>
   </property>
   <property name="text" >
    <string>Purge Volume</string>
   </property>
  </action>
  <action name="actionRelabelVolume" >
   <property name="icon" >
    <iconset resource="../main.qrc" >:/images/label.png</iconset>
   </property>
   <property name="text" >
    <string>Relabel Volume</string>
   </property>
  </action>
  <action name="actionAllVolumesFromPool" >
   <property name="icon" >
    <iconset resource="../main.qrc" >:/images/cartridge-edit.png</iconset>
   </property>
   <property name="text" >
    <string>Update all Volumes From Pool</string>
   </property>
   <property name="iconText" >
    <string>Update all Volumes From Pool</string>
   </property>
   <property name="toolTip" >
    <string>Update all Volumes From Pool</string>
   </property>
   <property name="statusTip" >
    <string>Update all Volumes From Pool</string>
   </property>
  </action>
  <action name="actionAllVolumes" >
   <property name="icon" >
    <iconset resource="../main.qrc" >:/images/cartridge-edit.png</iconset>
   </property>
   <property name="text" >
    <string>Update all Volumes from all Pools</string>
   </property>
   <property name="iconText" >
    <string>Update all Volumes from all Pools</string>
   </property>
   <property name="toolTip" >
    <string>Update all Volumes from all Pools</string>
   </property>
   <property name="statusTip" >
    <string>Update all Volumes from all Pools</string>
   </property>
  </action>
  <action name="actionVolumeFromPool" >
   <property name="icon" >
    <iconset resource="../main.qrc" >:/images/cartridge-edit.png</iconset>
   </property>
   <property name="text" >
    <string>Volume From Pool</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../main.qrc" />
 </resources>
 <connections/>
</ui>
